### ISSUE TITLE
codecData event for multiple inputs

### ISSUE THREAD

**Issue by codingquark** [fluent-ffmpeg/node-fluent-ffmpeg#486]
Currently, it seems to me that `codecData` is being fired only once and gives the result of the first input specified.

Is there any way I could get `codecData` output of all the inputs that have been specified?

**Comment by njoyard** [comment]
That's not currently possible. In a few weeks we'll implement new event handlers around ffmpeg output, so that will be made easier then!
Currently the only way I see is waiting for the 'end' or 'error' event to fire. Those receive ffmpeg output as their 2nd and 3rd arguments.

**Comment by codingquark** [comment]
Oh, OK. I will then make an ffprobe query on the media.

Thanks!

**Comment by njoyard** [comment]
Duplicate of #512
